### PR TITLE
Adding pod-qos.md in french

### PR DESCRIPTION
**Pull request by ShivangShandilya**
This PR fixes: #40515 

This PR translates pod-quos.md in French language

### CODE REVIEW

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Cours sur la qualité de service
+
+Kubernetes classe les Pods que vous exécutez et alloue chaque Pod dans un_quality de service (QoS) class_. Kubernetes utilise cette classification pour influencer la différenceLes gousses sont manipulées. Kubernetes effectue cette classification en fonction de la[demandes de ressources](/docs/concepts/configuration/manage-resources-containers/)du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avecle lien entre ces demandes et les limites de ressources.C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) classe. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressourceset les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour déciderquels Pods expulser d’un nœud expérimenté;[Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Le possibleLes classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources,Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de 'Burstable' etenfin des Pods 'garantis'
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:11]:
```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de qualité de service (QoS). Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+. Lorsque cette expulsion est due à la pression des ressources, seuls les Pods dépassentLes demandes de ressources sont candidates à l’expulsion.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:12]:
```suggestion

Lorsque qu'une expulsion est due à la pression des ressources, seuls les Pods sont dépassés. Les demandes de ressources sont candidates à l’expulsion.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Le Pod ne répond pas aux critères de la classe QoS 'Garantie'.
+- Au moins un conteneur dans le Pod a une demande ou une limite de mémoire ou de CPU.
+
+### Meilleur effort
+
+Les pods de la classe QoS « BestEfforts » peuvent utiliser des ressources de nœud qui ne sont pas spécifiquement affectéesaux Pods d’autres classes QoS. Par exemple, si vous avez un nœud avec 16 cœurs de processeur disponibles pour lekubelet, et vous attribuez 4 cœurs CPU à un Pod 'Guaranteed', puis un Pod dans le 'BestEffort'La classe QoS peut essayer d’utiliser n’importe quelle quantité des 12 cœurs de processeur restants.Le kubelet préfère expulser les pods 'BestEffort' si le nœud est soumis à une pression sur les ressources.
+
+#### Critères
+
+A Pod has a QoS class of `BestEffort` if it doesn't meet the criteria for either `Guaranteed`
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:44]: Not traduced

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+memory limit or a memory request, and none of the Containers in the Pod have a
+CPU limit or a CPU request.
+Containers in a Pod can request other resources (not CPU or memory) and still be classified as
+`BestEffort`.
+
+## Certains comportements sont indépendants de la classe QoS {#class-independent-behavior}
+
+Certains comportements sont indépendants de la classe QoS attribuée par Kubernetes. Par exemple:
+
+- Tout conteneur dépassant une limite de ressources sera tué et redémarré par le kubelet sansaffectant d’autres conteneurs dans ce pod.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:55]:
```suggestion
- Tout conteneur dépassant une limite de ressources sera tué et redémarré par Kubelet sans affecter les autres conteneurs dans ce pod.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- Si un conteneur dépasse sa demande de ressources et que le nœud sur lequel il s’exécute fait facepression des ressources, le Pod dans lequel il se trouve devient un candidat pour [expulsion](/docs/concepts/scheduling-eviction/node-pressure-eviction/).Si cela se produit, tous les conteneurs du pod seront résiliés. Kubernetes peut créer unPod de remplacement, généralement sur un nœud différent.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:56]:
```suggestion
- Si un conteneur dépasse sa demande de ressources et que le nœud sur lequel il s’exécute fait face a une pression sur ses ressources, le Pod dans lequel il se trouve devient un candidat pour [eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Si cela se produit, tous les conteneurs du pod seront tués. Kubernetes peut créer un Pod de remplacement sur un nœud différent.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+- La demande de ressource d’un Pod est égale à la somme des demandes de ressources deses conteneurs de composants et la limite de ressources d’un pod est égale à la somme deles limites de ressources de ses conteneurs composants.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:57]: Incomprehensible, reformulate.

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- Le planificateur kube ne tient pas compte de la classe QoS lors de la sélection des Pods à[préempter](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption).La préemption peut se produire lorsqu’un cluster ne dispose pas de suffisamment de ressources pour exécuter tous les Podsvous avez défini.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:58]:
```suggestion
- Le planificateur kube ne tient pas compte de la classe QoS lors de la sélection des Pods à [préempter](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption). La préemption peut se produire lorsqu’un cluster ne dispose pas de suffisamment de ressources pour exécuter tous les Pods que vous avez défini.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
+---
+titre: Classes de qualité de service des pods
+content_type: concept
+poids: 85
+---
+
+Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en conséquence des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:7]:
```suggestion
Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```

**Review by perriea — CHANGES_REQUESTED** [review]
First check

File: content/fr/docs/concepts/workloads/pods/pod-qos.md (hunk 6 above)
Comment (ShivangShandilya) [reply content/fr/docs/concepts/workloads/pods/pod-qos.md:58]: I checked with the English version of the same and translating the above text in English is exactly similar to the English version. The translation here seems correct to me but do correct me.

English Version: https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/pods/pod-qos.md

File: content/fr/docs/concepts/workloads/pods/pod-qos.md (hunk 3 above)
Comment (ShivangShandilya) [reply content/fr/docs/concepts/workloads/pods/pod-qos.md:45]: Resolved

File: content/fr/docs/concepts/workloads/pods/pod-qos.md (hunk 6 above)
Comment (perriea) [reply content/fr/docs/concepts/workloads/pods/pod-qos.md:53]: I look this tonight

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## Cours sur la qualité de service
+
+Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de qualité de service (QoS). Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:11]:
```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de QoS. Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
+---
+titre: Classes de qualité de service des pods
+content_type: concept
+poids: 85
+---
+
+Cette page présente les classes de qualité de service (QoS) dans Kubernetes et explique comment Kubernetes affecte une classe QoS à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:7]:
```suggestion
Cette page présente les classes de Quality of Service (QoS, "Qualité de service") dans Kubernetes et explique comment Kubernetes affecte une classe à chaque Pods en fonction des contraintes de ressources que vous spécifiez pour les conteneurs dans ce Pod. Kubernetes s’appuie sur cette classification pour prendre des décisions sur les Pods à supprimer lorsqu’il n’y a pas assez de ressources disponibles sur un nœud.
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+## Cours sur la qualité de service
```
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:9]:
```suggestion
## Explication du fonctionnement de qualité de service
```

File: content/fr/docs/concepts/workloads/pods/pod-qos.md (hunk 9 above)
Comment (perriea) [inline content/fr/docs/concepts/workloads/pods/pod-qos.md:11]:
```suggestion
Kubernetes classe les Pods que vous exécutez et affecte chaque Pod dans une classe de QoS. Kubernetes utilise cette classification pour influencer les différents pools sont manipulées. Kubernetes effectue cette classification en fonction de la [demandes de ressources](/docs/concepts/configuration/manage-resources-containers/) du {{< glossary_tooltip text="Containers » term_id="container » >}} dans ce Pod, avec le lien entre ces demandes et les limites de ressources. C’est ce qu’on appelle {{< glossary_tooltip text="Quality of Service » term_id="qos-class » >}}(QoS) class. Kubernetes attribue à chaque Pod une classe QoS en fonction des demandes de ressources et les limites de ses conteneurs constitutifs. Les classes QoS sont utilisées par Kubernetes pour décider quels Pods expulser d’un nœud; [Pression du nœud](/docs/concepts/scheduling-eviction/node-pressure-eviction/). Les classes QoS sont `Guaranteed`, `Burstable` et `BestEffort`. Lorsqu’un nœud est à court de ressources, Kubernetes supprimera d’abord les pods `BestEffort` exécutés sur ce nœud, suivis de `Burstable` et enfin des Pods `Guaranteed`.
```

**Review by perriea — CHANGES_REQUESTED** [review]
Second check

**Comment by ShivangShandilya** [comment]
Looks good to me now. Would you like to do a third check @perriea?

**Review by perriea — APPROVED** [review]

**Comment by perriea** [comment]
@remyleone what's the command to relaunch tests ? `/retest` nope ...  `/test all` nope

**Comment by remyleone** [comment]
@perriea I've used those commands before: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md

**Comment by perriea** [comment]
@sftim @onlydole an idea ? Any command tested don't work :/

**Comment by sftim** [comment]
Prow doesn't let you trigger a rebuild I'm afraid. If you edit your last commit and re-push, that can help. The underlying issue is tracked as https://github.com/kubernetes/website/issues/35780

**Comment by perriea** [comment]
> Prow doesn't let you trigger a rebuild I'm afraid. If you edit your last commit and re-push, that can help. The underlying issue is tracked as #35780

Thanks ! @ShivangShandilya can you re-push your latest commit ?

**Comment by ShivangShandilya** [comment]
> > Prow doesn't let you trigger a rebuild I'm afraid. If you edit your last commit and re-push, that can help. The underlying issue is tracked as #35780
> 
> Thanks ! @ShivangShandilya can you re-push your latest commit ?

Done 😄

**Comment by aj11anuj** [comment]
Hey @ShivangShandilya, netlify checks are still failing.. you need to re-push again.

**Comment by ShivangShandilya** [comment]
I don't know why the checks are failing
If everything looks fine, let's merge this ASAP.
It's been open for a while now

**Comment by reylejano** [comment]
@ShivangShandilya , this won't merge because checks are failing
Can squash your commits to one commit, update the commit message, then re-push. That should re-trigger the checks.

**Comment by natalisucks** [comment]
@ShivangShandilya Hey there, just checking that you've seen [Rey's comment above](https://github.com/kubernetes/website/pull/40591#issuecomment-1890334769) about how to re-trigger the tests, that way this PR will merge as it has the appropriate approvals – thanks!

**Comment by divya-mohan0209** [comment]
@ShivangShandilya Thank you for your contributions! Since there has been no activity on this PR in ~2 months, I will be closing this off. Please note that in case you wish to resume work on the PR, you can do so by reopening it & following Rey's comments earlier.
/close

**Pull request closed without merging**